src/Uploadcare/Signature/SecureSignature.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Uploadcare\Signature;
+
+use Exception;
+use DateTime;
+
+/**
+ * Class SecureSignature
+ * @package Uploadcare
+ */
+class SecureSignature implements SignatureInterface
+{
+    /**
+     * @var string
+     */
+    private $signature;
+
+    /**
+     * @var DateTime
+     */
+    private $expire;
+
+    /**
+     * SecureSignature constructor.
+     *
+     * @param string $secretKey
+     * @param int $expireTimeInSeconds
```

Comment:
expiretimeinseconds seems to elaborate var name.
if you need more context, doc string is there to help.
```suggestion
     * @param int $lifetime
```

Reply:
Thank you! Fixed

Reply:
change the name everywhere, it doesn't make sence